### DISCUSSION TITLE
Syslog and Kibana Elastic

### DISCUSSION THREAD

**Discussion by tuursix** [Security-Onion-Solutions/securityonion#8921] [General]
If you want to use your Security onion as a Syslog Server, do you have to install filebeat or syslog-ng on the different devices that needs to send syslog to Security onion? (I already put the did sudo so-allow and chose syslog and type the IP) How does Kibana Elastic work I always had a bron IP from my network, but the destination address is always a multicast or broadcast ip address. Can someone help me?

**Comment by reyesj2** [comment]
Are you sending logs to a sensor as a part of a distributed deployment? If so check out https://docs.securityonion.net/en/2.3/firewall.html#allow-hosts-to-send-syslog-to-a-sensor-node

Otherwise, check the network connection between SO and your devices.

Did you configure syslog on your end device to forward logs to the SO server IP?
